### PR TITLE
fixed course name used in student repo links

### PR DESCRIPTION
**Pull request by 0xf8f8ff**
To be updated with similar repository links from the lab views.

### CODE REVIEW

File: public/src/pages/TeacherPage.tsx [hunk 1]
```diff
@@ -397,6 +400,15 @@ export class TeacherPage extends ViewPage {
         return [];
     }
 
+    private async getCourseCodeURL(courseID: number): Promise<string> {
```
[the remark this answers is not in the document]
Comment (0xf8f8ff) [reply public/src/pages/TeacherPage.tsx]: Done

File: public/src/components/teacher/GroupResults.tsx [hunk 2]
```diff
@@ -3,9 +3,11 @@ import { Assignment, Course } from "../../../proto/ag_pb";
 import { DynamicTable, Row, Search, StudentLab } from "../../components";
 import { IAssignmentLink, IStudentSubmission } from "../../models";
 import { sortByScore } from "./sorter";
+import { slugify } from "../../helper"
 
 interface IResultsProps {
     course: Course;
+    courseCodeURL: string;
```
[the remark this answers is not in the document]
Comment (0xf8f8ff) [reply public/src/components/teacher/GroupResults.tsx]: And done

File: public/src/pages/views/UserView.tsx [hunk 3]
```diff
@@ -167,7 +167,7 @@ export class UserView extends React.Component<IUserViewerProps, IUserViewerState
 
     // return link to github account if there is no course information, otherwise return link to the student labs repo
     private repoLink(user: string): string {
-        const repoLink = "https://github.com/" + this.props.courseCode + "/" + user + "-labs";
+        const repoLink = this.props.courseCode + user + "-labs";
```
Comment (meling) [inline public/src/pages/views/UserView.tsx]: Should also be courseURL

Comment (meling) [reply public/src/pages/views/UserView.tsx]: Can we reuse the generate function somehow?

**Pull request merged**